docs/backbone.radio.md

```diff
@@ -0,0 +1,131 @@
+# Backbone Radio
+
+The Backbone Radio provides easy support for a number of messaging patterns for
+Backbone and Marionette. This is provided through two basic constructs:
+
+* Events - trigger events on a global object
+* Requests - a global request/reply implementation
+
+
+Radio takes these two constructs and adds the channel implementation - providing
+namespaces for events and requests. In short, Radio is a global, namespaced,
+message bus system designed to allow two otherwise unrelated objects to
+communicate and share information.
+
+Let's look at a simplified example in Marionette:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var NotificationView = Mn.View.extend({
+  channelName: 'notify',
```

Comment:
Awesome, cheers @paulfalgout - I didn't see `radioEvents` or `radioRequests` in the source code. I'll get all that changed over :+1: